### PR TITLE
Examples for Central Difference Time Integration in MASTODON

### PR DESCRIPTION
**Pull request by mriganabh**
I noticed that the images and plots were not added to my commit. I tried attaching them to this file but it does not seem to work. Let me know how should send them.

### CODE REVIEW

File: examples/ex10a/MASTODON_Explicit_HEX8.i [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+  [./strain]
+    type = ComputeSmallStrain
+    displacements = 'disp_x disp_y disp_z'
+    implicit = false
+  [../]
+  [./stress]
+    type = ComputeLinearElasticStress
+    implicit = false
+  [../]
+[]
```
Comment (cbolisetti) [inline examples/ex10a/MASTODON_Explicit_HEX8.i:134]: You only need implicit = false in the strain block

File: examples/ex10a/MASTODON_Explicit_HEX8.i [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+    implicit = false
+  [../]
+  [./disp_z]
+    type = DirichletBC
+    displacements = 'disp_x disp_y disp_z'
+    variable = disp_z
+    boundary = 'back'
+    value = 0
+    implicit = false
+  [../]
```
Comment (cbolisetti) [inline examples/ex10a/MASTODON_Explicit_HEX8.i:167]: Please remove the implicit = false line

File: examples/ex10a/MASTODON_Explicit_HEX8.i [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+[]
+
+[Functions]
+[]
```
Comment (cbolisetti) [inline examples/ex10a/MASTODON_Explicit_HEX8.i:171]: remove

File: examples/ex10a/MASTODON_Explicit_HEX8.i [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+  [../]
+   [./stress_zz_20]
+    type = PointValue
+    variable = stress_zz
+    point = '0.0 0.0 20'
+  [../]
+[]
+
+[VectorPostprocessors]
+[]
```
Comment (cbolisetti) [inline examples/ex10a/MASTODON_Explicit_HEX8.i:231]: Remove

File: examples/ex10a/MASTODON_Explicit_HEX8.i [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+  [../]
+  [./stress_zz]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./layer_id]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+
```
Comment (cbolisetti) [inline examples/ex10a/MASTODON_Explicit_HEX8.i:46]: Remove empty line

File: examples/ex10a/MASTODON_Explicit_HEX8.i [hunk 6]
```diff
@@ -0,0 +1,241 @@
...
+[]
+
+
```
Comment (cbolisetti) [inline examples/ex10a/MASTODON_Explicit_HEX8.i:49]: Remove empty line

File: examples/ex10a/MASTODON_Implicit_HEX8.i [hunk 7]
```diff
@@ -0,0 +1,263 @@
+[Mesh]
```
Comment (cbolisetti) [inline examples/ex10a/MASTODON_Implicit_HEX8.i:1]: Same comments as above for this file.

File: examples/ex10b/MASTODON_Explicit_HEX20.i [hunk 8]
```diff
@@ -0,0 +1,250 @@
+[Mesh]
```
Comment (cbolisetti) [inline examples/ex10b/MASTODON_Explicit_HEX20.i:1]: Same comments as above. Please remove extra empty lines and empty blocks. 
implicit = false is only needed in the strain block. Please remove it from other blocks.

**Comment by milljm** [comment]
You might consider rebasing off upstream/devel to get rid of those pesky merge commits.

**Comment by mriganabh** [comment]
I am closing this pull request due to the pesky commits. I have created a new branch "mboruah" which is free from these unnecessary commits and will scrap this branch.

**Pull request closed without merging**